pkg/v1workermeta/api_test.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1workermeta
+
+import (
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	. "github.com/pingcap/check"
+
+	"github.com/pingcap/dm/dm/pb"
+	"github.com/pingcap/dm/pkg/terror"
+	"github.com/pingcap/dm/pkg/utils"
+)
+
+type testAPI struct{}
+
+var _ = Suite(&testAPI{})
+
+func (t *testAPI) TestAPI(c *C) {
+	_, currFile, _, _ := runtime.Caller(0)
+	srcMetaPath := filepath.Join(filepath.Dir(currFile), "v106_data_for_test")
+	srcDBPath := filepath.Join(srcMetaPath, "kv")
+
+	oldMetaPath := metaPath
+	oldDBPath := dbPath
+	defer func() {
+		metaPath = oldMetaPath
+		dbPath = oldDBPath
+	}()
+
+	metaPath = c.MkDir()
+	dbPath = filepath.Join(metaPath, "kv")
+
+	// copy test data to a temp directory.
+	copyDir(c, srcDBPath, dbPath)
```

Comment:
seems go prefer `copy(dst, src)`

Reply:
updated in https://github.com/pingcap/dm/pull/846/commits/a638dcdc19d0311a4af8e42eec4279849b0eb8b3.